WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -143,6 +143,7 @@
         <%= simple_form_for @user do |f| %>
           <%= hidden_field_tag "section", "email" %>
           <%= f.input :email, disabled: !editable_fields.include?(:email) %>
```

Comment:
If you change this line as follows:

```suggestion
           <%= f.input :email, disabled: !editable_fields.include?(:email), hint: t('.confirm_new_email') %>
```

Can you get rid of the `span.help-block` you added, along with the CSS you added above?